packages/lib/src/FormController/FormController.ts

```diff
@@ -289,6 +291,13 @@ export class FormController {
     return fieldValues.some((field: FormField) => field.instance !== null && field.meta.isTouched);
   }
 
+  //where any of the form fields ever changed
+  @computed
+  get isChanged(): boolean {
```

Comment:
isn't it the combination of isTouched + !isDirty ? Maybe we don't even need additional derived state in the API? WDYT?

Reply:
for example: isChanged should be `true` in case isTouched is `true` and isDirty is `false` and it can be checked in userland. Am I mistaken?